vectorlm/trainer.py

```diff
@@ -291,9 +334,14 @@ def eval_step(self, epoch: int) -> float:
             with torch.no_grad():
                 batch.pop("id")
                 batch["input_ids"] = batch["input_ids"].type(torch.LongTensor)
+                num_tokens = len(batch["input_ids"].flatten())
                 batch["labels"] = batch["labels"].type(torch.LongTensor)
-                out = self.model(**batch)
-                eval_loss += out.loss
+                batch = {k: v.to(torch.cuda.current_device()) for k, v in batch.items()}
```

Comment:
As above, I believe this line is unnecessary.

Reply:
Resolved by deleting this line since we are exclusively running FSDP.